packages/cyberstorm/src/components/Layout/Teams/TeamsLayout.tsx

```diff
@@ -13,7 +12,7 @@ import { PageHeader } from "../BaseLayout/PageHeader/PageHeader";
 import { useDapper } from "@thunderstore/dapper";
 import usePromise from "react-promise-suspense";
 
-//TODO: Use Alert component
```

Comment:
The component doesn't seem to use Alert component yet, was this comment intentionally removed?

Reply:
Alert has been added to the subcomponent, whoever did that probably forgot to remove the TODO comment here.